src/web/lib/getZIndex.test.tsx

```diff
@@ -2,9 +2,10 @@ import { getZIndex } from './getZIndex';
 
 describe('getZIndex', () => {
 	it('gets the correct zindex for group and sibling', () => {
-		expect(getZIndex('banner')).toBe('z-index: 13;');
-		expect(getZIndex('dropdown')).toBe('z-index: 12;');
-		expect(getZIndex('burger')).toBe('z-index: 11;');
+		expect(getZIndex('banner')).toBe('z-index: 14;');
```

Comment:
Aside (not relevant to this PR): Are these tests adding much value? They need updating each time a new index is added which screams implementation details to me.

Ideally, we'd have a test that renders a component that uses this function under the hood and then asserts based on visibility?

Reply:
Good point! It might be worth adding this as a health week ticket